distribution/ddf-common/src/main/resources/etc/log4j2.config.xml

```diff
@@ -12,7 +12,7 @@
  *
  **/
 -->
-<Configuration status="WARN">
+<Configuration status="OFF">
```

Comment:
Changing status to off here should suppress an ERROR from log4j2 about the configuration file.